### PR TITLE
Show validation errors in drawer-item

### LINKED ISSUE #16091: Validation errors do not show in O2M drawer
### Describe the Bug

Validation errors no longer appear in the drawer. 
For example when creating an item in a O2M list. If a required field is not filled, the validation error does not appear. This is occuring on the main branch and not on 9.18.1

### To Reproduce

1. Use latest code from main branch
2. Create a collection team 
3. Create a collection players. Add a required field 'Name..  Add a M2O field for team. 
4. Add an O2M field for players in the team collection
5. Add a new team
6. In the O2M list of the team creation page, click 'Create New' so that the drawer opens to create a new player
7. Leave the required 'name' field blank and click save

Result : No errors are shown and drawer does not close

### Errors Shown

An validation error should show, but it does not

### What version of Directus are you using?

latest main branch

### What version of Node.js are you using?

18

### What database are you using?

SQLite 5.12.0

### What browser are you using?

Chrome

### How are you deploying Directus?

locally

### PR DESCRIPTION
**Pull request by Nitwel**
By splitting up the `nested` prop into to separate, we can disable the `no visible fields` info but keep the validation errors.

Fixes #16091

### CODE REVIEW

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]
Lets flip it from `hideX` to `showX` with the inversed default values to stay consistent with other props like this in other components

**Comment by Nitwel** [comment]
Haha, had it that way and flipped it to hideX so their name matches their default value.
E.g. `<v-table hideX>` is more direct that `<v-table :showX="false">` and thus you might think `<v-table showX>` would imply that X is hidden by default. Thoughts?

**Comment by rijkvanzanten** [comment]
@Nitwel I'd rather keep it consistent across the codebase 🙂 There's also that tiny opinionated code quality thing where you should prevent double negatives (eg "Do X = true" is easier to mentally parse than "Don't do X = false")

**Review by rijkvanzanten — APPROVED** [review]

File: app/src/components/v-form/v-form.vue [hunk 1]
```diff
@@ -1,13 +1,13 @@
 <template>
 	<div ref="el" class="v-form" :class="gridClass">
 		<validation-errors
-			v-if="!nested && validationErrors.length > 0"
+			v-if="!showValidationErrors && validationErrors.length > 0"
```
Comment (paescuj) [inline app/src/components/v-form/v-form.vue:4]:
```suggestion
			v-if="showValidationErrors && validationErrors.length > 0"
```

File: app/src/components/v-form/v-form.vue [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 			:validation-errors="validationErrors"
 			:fields="fields ? fields : []"
 			@scroll-to-field="scrollToField"
 		/>
 		<v-info
-			v-if="noVisibleFields && !nested && !loading"
+			v-if="noVisibleFields && !showNoVisibleFields && !loading"
```
Comment (paescuj) [inline app/src/components/v-form/v-form.vue:10]:
```suggestion
			v-if="noVisibleFields && showNoVisibleFields && !loading"
```

File: app/src/components/v-form/v-form.vue [hunk 3]
```diff
@@ -203,7 +205,7 @@ const noVisibleFields = computed(() => {
 watch(
 	() => props.validationErrors,
 	(newVal, oldVal) => {
-		if (props.nested) return;
+		if (props.showValidationErrors) return;
```
Comment (paescuj) [inline app/src/components/v-form/v-form.vue:208]:
```suggestion
		if (!props.showValidationErrors) return;
```

**Review by paescuj — APPROVED** [review]
LGTM, besides a small clean-up due to flipped logic.

**Pull request merged**